src/components/scenes/ChangePinScene.tsx

```diff
@@ -90,38 +90,29 @@ const ChangePinSceneComponent = ({
             onChangePin={handleChangePin}
           />
         </EdgeAnim>
-        <EdgeAnim
-          style={styles.actions}
-          visible={isValidPin}
-          enter={{ type: 'fadeInDown' }}
-          exit={{ type: 'fadeOutDown' }}
-        >
-          <MainButton
-            label={mainButtonLabel}
-            type="primary"
-            onPress={handlePress}
-          />
-        </EdgeAnim>
+        <SceneButtons
+          primary={{
+            label: mainButtonLabel,
+            onPress: handlePress,
+            disabled: !isValidPin
+          }}
+          animDistanceStart={1}
```

Comment:
Yes. Enforcing consistency. We either have all buttons visible above the keyboards or have all buttons pinned to the bottom. 

The existing UI has a mix of both, and the limiting factor was the password input design, which has always presented problems. The ultimate solution there was to just pin to the bottom and not bother to try to keep buttons visible above the keyboard. Because of this limiting factor, I changed all the scenes to match.

Reply:
Because the keyboard is almost always occluding the Next button, I'll concede. But I think consistency trumping an elegant UI needs to be carefully balanced. It's not an all or nothing.

Reply:
The conflict we tend to have is that I (and Madison) think consistency is not at odds with, but a _pillar_ of elegant UI. Consistency makes things look clean, whereas inconsistency makes things look messy, unintentional, and unrefined.